demo_drivers/bifurcation_tracking/adaptive_hopf.cc

```diff
@@ -1304,7 +1304,7 @@ int main()
   {
    Vector<std::complex<double> > eigenvalues;
    //Now solve the eigenproblem
-   problem.solve_eigenproblem(6,eigenvalues,eigenvectors);
+   problem.solve_eigenproblem_legacy(6,eigenvalues,eigenvectors);
```

Comment:
'fraid so because there was one more interface (the one that only returns the eigenvalues) that can't be distinuished otherwise. It's on your list already: everything that's contains the string _legacy should go.

  Done.